Serhii_Popov/Week2/hw_4.3.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+class testreplace(unittest.TestCase):
+    def test_replace(self):
+        with self.assertRaises(TypeError):
+           replacemaxmin ('2')
+           #replacemaxmin(print([10, 20, 30, 40, 50, 100, 150]))
+        test_list = [10, 20, 30, 150, 40, 50, 100]
+        self.assertEqual(replacemaxmin(test_list), [150, 20, 30, 10, 40, 50, 100])
+#arr = [10, 20, 30, 150, 40, 50, 100]
+
+
+
+def replacemaxmin(arr):
+    top = (arr.index(max(arr)))
+    bot = (arr.index(min(arr)))
+    for el in range(len(arr)):
```

Comment:
dont use loop

find minIndex, find maxIndex
than using above results try to switch element with indexes